asr/review.py

```diff
@@ -231,14 +232,16 @@ def review(dataset,
         # TODO: save results.
         print('\nClosing down the automated systematic review.')
         print('\nSaving results.')
+        if reviewer.log_file:
+            reviewer.save_logs(reviewer.log_file)
 
     # save the result to a file
     if reviewer.log_file:
         reviewer.save_logs(reviewer.log_file)
 
     # print the results
-    if reviewer.verbose:
-        print(reviewer._logger._print_logs())
+#     if reviewer.verbose:
```

Comment:
Keep this in?

Reply:
Actually, I had something else here, but something went wrong with the merging/rebasing..  I removed it because the log files are getting too big. It completely swamps the console/terminal making stuff hard to find.